src/carplay.cs

```diff
@@ -588,8 +600,14 @@ interface CPListSection : NSSecureCoding {
 		[NullAllowed, Export ("sectionIndexTitle")]
 		string SectionIndexTitle { get; }
 
+#if !XAMCORE_5_0
 		[Export ("items", ArgumentSemantic.Copy)]
+		[Obsolete ("Use 'Items : ICPListTemplateItem []' instead.")]
 		CPListItem [] Items { get; }
+#endif
+
+		[Export ("items", ArgumentSemantic.Copy)]
+		ICPListTemplateItem [] Items { get; }
```

Comment:
As I expected, having two properties with the same name results in an error:
```
carplay.cs(610,26): error CS0102: The type 'CPListSection' already contains a definition for 'Items'
```
The following should fix the issue:
```suggestion
#if !XAMCORE_5_0
		[Sealed]
		[Export ("items", ArgumentSemantic.Copy)]
		[Obsolete ("Use 'Items2 : ICPListTemplateItem []' instead.")]
		CPListItem [] Items { get; }
#eendif

		[Export ("items", ArgumentSemantic.Copy)]
#if !XAMCORE_5_0
		ICPListTemplateItem [] Items2 { get; }
#else
		ICPListTemplateItem [] Items { get; }
#endif
```
That means we have Items2 but in the future it will be Items.